misc/docs-linter/links.go

```diff
@@ -21,6 +21,20 @@ func extractLocalLinks(fileContent []byte) []string {
 	for scanner.Scan() {
 		line := scanner.Text()
 
+		// Find embedmd links
+		if strings.Contains(line, "[embedmd]") {
+			openPar := strings.Index(line, "(")
```

Comment:
Is it possible to have nested parentheses? I don't think this would work correctly if it is possible.

Reply:
[767d375](https://github.com/gnolang/gno/pull/2416/commits/767d3753e6260a750bf8a03a80a6bddc827626ed) & [1ae1b0b](https://github.com/gnolang/gno/pull/2416/commits/1ae1b0bb2843ba8317457f985d02041e834e5fa8)

Reply:
Added some more checks: [de8cc5b](https://github.com/gnolang/gno/pull/2416/commits/de8cc5b5cde2258bba16571e2b8682b1eacd3910)